### PR TITLE
Action to deal with custom fonts

### PR DESCRIPTION
**Pull request by danixeee**
Everything is in an action readme...

Tested on:

- [x] macOS
- [x] windows

### CODE REVIEW

**Comment by danixeee** [comment]
@ivansenic Tested both on windows and macOS... I have commented on a code that is called on post-render action to uninstall fonts. Should we keep it commented or remove uninstall action for now?

File: packages/nexrender-action-fonts-plainly/src/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const os = require('os');
+const {dirname} = require('path');
+
+const isWin = os.type() === 'Windows_NT';
```
Comment (ivansenic) [inline packages/nexrender-action-fonts-plainly/src/index.js:4]: will this also work on all Windows machines?

**Review by ivansenic — APPROVED** [review]
Seems also very good.. Only concern is this `Windows_NT` and if this would work on the other Win machines?

**Comment by danixeee** [comment]
@ivansenic [Official docs](https://nodejs.org/api/os.html#os_os_type) says:

> Returns the operating system name as returned by uname(3). For example, it returns 'Linux' on Linux, 'Darwin' on macOS, and 'Windows_NT' on Windows.

So, `Windows_NT` is for all windows systems.

File: packages/nexrender-action-fonts-plainly/src/fontsWin.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const { registryAdd, registryExists, registryRemove} = require('./reg');
+const { basename, join } = require('path');
+
+const FONTS_DIRECTORY = "C:\\windows\\fonts";
+const FONTS_REG_KEY = "HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts";
+
+const install = async (workDir, fonts) => {
+  return Promise.all(
+    fonts.map(async (f) => {
+      const fontName = basename(f);
```
[the remark this answers is not in the document]
Comment (danixeee) [reply packages/nexrender-action-fonts-plainly/src/fontsWin.js:11]: Logs seem helpful, I had some code and permission errors and logs helped me to resolve them.

**Pull request merged**